pom.xml

```diff
@@ -119,6 +120,11 @@
             <artifactId>msgpack-core</artifactId>
             <version>0.8.24</version>
         </dependency>
+        <dependency>
```

Comment:
Do we need this dependency? According to https://spring.io/blog/2021/12/10/log4j2-vulnerability-and-spring-boot?s=09 I think we need to only set the above `log4j2.version` property.

Reply:
You might be right. I am not good enough on spring magic...
Corrected